cmd/ttn-lw-stack/commands/config.go

```diff
@@ -53,22 +55,24 @@ type Config struct {
 	DTC              devicetemplateconverter.Config    `name:"dtc"`
 	QRG              qrcodegenerator.Config            `name:"qrg"`
 	PBA              packetbrokeragent.Config          `name:"pba"`
+	DeviceRepository devicerepository.Config           `name:"device-repository"`
 	OutputFormat     string                            `name:"output-format" yaml:"output-format" description:"Output format"`
 }
 
 // DefaultConfig contains the default config for the ttn-lw-stack binary.
 var DefaultConfig = Config{
-	ServiceBase: shared.DefaultServiceBase,
-	IS:          shared_identityserver.DefaultIdentityServerConfig,
-	GS:          shared_gatewayserver.DefaultGatewayServerConfig,
-	NS:          shared_networkserver.DefaultNetworkServerConfig,
-	AS:          shared_applicationserver.DefaultApplicationServerConfig,
-	JS:          shared_joinserver.DefaultJoinServerConfig,
-	Console:     shared_console.DefaultConsoleConfig,
-	GCS:         shared_gatewayconfigurationserver.DefaultGatewayConfigurationServerConfig,
-	DTC:         shared_devicetemplateconverter.DefaultDeviceTemplateConverterConfig,
-	QRG:         shared_qrcodegenerator.DefaultQRCodeGeneratorConfig,
-	PBA:         shared_packetbrokeragent.DefaultPacketBrokerAgentConfig,
+	ServiceBase:      shared.DefaultServiceBase,
+	IS:               shared_identityserver.DefaultIdentityServerConfig,
+	GS:               shared_gatewayserver.DefaultGatewayServerConfig,
+	NS:               shared_networkserver.DefaultNetworkServerConfig,
+	AS:               shared_applicationserver.DefaultApplicationServerConfig,
+	JS:               shared_joinserver.DefaultJoinServerConfig,
+	Console:          shared_console.DefaultConsoleConfig,
+	GCS:              shared_gatewayconfigurationserver.DefaultGatewayConfigurationServerConfig,
+	DTC:              shared_devicetemplateconverter.DefaultDeviceTemplateConverterConfig,
+	QRG:              shared_qrcodegenerator.DefaultQRCodeGeneratorConfig,
+	PBA:              shared_packetbrokeragent.DefaultPacketBrokerAgentConfig,
+	DeviceRepository: shared_devicerepository.DefaultDeviceRepositoryConfig,
```

Comment:
Inconsistent; must be DR

Reply:
There is existing (but previously unused) `device-repository.*` config, so renaming that to `dr` is typically a breaking change. Personally, I like `DR` better as well, is it ok to change it in this case?

Reply:
You can also rename the field, but leave the tag as `device-repository`. Maybe even setup an alias?

Reply:
Indeed. This refers to `DeviceRepository` here in code

Anyway, I'm fine with deprecating the old config. It wasn't used anyway. But if you can keep it compatible, I'm good with that too.

Reply:
Since this config was never used or documented, I don't think we violate compatibility by changing it to `dr`.

Reply:
This is not addressed @neoaggelos